released/packages/coq-vellvm/coq-vellvm.1.0.20240416/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+name: "coq-vellvm"
+maintainer: "stevez@cis.upenn.edu"
+synopsis: "Coq library implementing (executable) semantics for LLVM IR"
+
+homepage: "https://github.com/vellvm/vellvm"
+dev-repo: "git+https://github.com/vellvm/vellvm.git"
+bug-reports: "https://github.com/vellvm/vellvm/issues"
+authors: [
+  "Steve Zdancewic <stevez@cis.upenn.edu>"
+  "Yannick Zakowski <yannick.zakowski@inria.fr>"
+  "Calvin Beck <hobbes@seas.upenn.edu>"
+  "Irene Yoon <euisuny@cis.upenn.edu>"
+  "Gary (Hanxi) Chen <hanxic@seas.upenn.edu>"
+]
+license: "GPL-3.0-or-later"
```

Comment:
@Chobbes just to flag up, this license means that it will be illegal to distribute compiled `.vo` files for the parts of Vellvm that use Mathematical Components (`coq-mathcomp-ssreflect`). The problem is that according to the Free Software Foundation, the CECILL-B license used by MathComp is incompatible with GPL. Read more about it here: https://www.gnu.org/licenses/license-list.html#CeCILL-B

Reply:
Hmmm. That's annoying. I'll ping @Zdancewic about it and maybe we can look into either updating the license or pulling the `coq-mathcomp-ssreflect` dependency (I'm honestly not sure what we use it for).